solr_conf/conf/solrconfig.xml

```diff
@@ -158,16 +169,27 @@
     </lst>
   </requestHandler>
 
+
   <requestDispatcher handleSelect="true" >
     <requestParsers enableRemoteStreaming="true" multipartUploadLimitInKB="2048" />
   </requestDispatcher>
 
   <requestHandler name="/analysis/field" startup="lazy" class="solr.FieldAnalysisRequestHandler" />
   <requestHandler name="/update" class="solr.UpdateRequestHandler"  />
+
   <updateHandler class="solr.DirectUpdateHandler2">
     <updateLog>
       <str name="dir">${solr.data.dir:}</str>
     </updateLog>
+    <autoCommit>
```

Comment:
It was copied from Plano's configuration file and are possibly set by
jetty.

*Weiwei Shi*

Digital Initiative Applications Librarian
University of Alberta Libraries
2-10L Cameron Library
Edmonton, Alberta, Canada T6G 2J8
Phone:(780)492-7802
Fax: (780)248-1209
Email: weiwei.shi@ualberta.ca

On Fri, Sep 4, 2015 at 10:25 AM, pgwillia <notifications@github.com> wrote:

> In solr_conf/conf/solrconfig.xml
> <https://github.com/ualbertalib/HydraNorth/pull/559#discussion_r38768805>:
>
> >    <updateHandler class="solr.DirectUpdateHandler2">
> >      <updateLog>
> >        <str name="dir">${solr.data.dir:}</str>
> >      </updateLog>
> > +    <autoCommit>
>
> Just curious how the autocommit values were chosen?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/pull/559/files#r38768805>.
>